src/screens/tabs/Profile/screens/CreateEditAccount/index.js

```diff
@@ -9,33 +10,56 @@ import UserIcon from '@/commonComponents/UserIcon';
 import RainbowButton from '@/components/buttons/RainbowButton';
 import Text from '@/components/common/Text';
 import { FontStyles } from '@/constants/theme';
-import useAccounts from '@/hooks/useAccounts';
+import {
+  createSubaccount,
+  editSubaccount,
+  importAccountFromPem,
+} from '@/redux/slices/keyring';
 
 import EditEmoji from '../EditEmoji';
 import styles from './styles';
 
-const CreateEditAccount = ({ modalRef, account, accountsModalRef }) => {
+/**
+ * @param {{ modalRef: any, accountsModalRef?: any, account?: any, pem?: string, createImportModalRef?: any } param
```

Comment:
define the real interfaces here, or we could move the file to TS, maybe its not that hard

Reply:
I was on it!